content/docs/services/cloud-gov-identity-provider.md

```diff
@@ -4,7 +4,7 @@ menu:
     parent: services
 title: cloud.gov identity provider
 name: "cloud-gov-identity-provider"
-description: "Manage client credentials for authenticating cloud.gov users in your app"
+description: "Authenticate cloud.gov users in your app"
```

Comment:
I wonder if we can find a more plain-language way of saying this - not sure if I'd understand "Authenticate cloud.gov users in your app" at first read if I didn't know what this service was. Maybe "Use cloud.gov authentication in your application as a login system for your users"?